content/ngf/how-to/data-plane-configuration.md

```diff
@@ -11,83 +11,205 @@ Learn how to dynamically update the NGINX Gateway Fabric global data plane confi
+  namespace: default
+spec:
+  telemetry:
+    exporter:
+      endpoint: "my.telemetry.collector:9000"
+      interval: "60s"
+      batchSize: 20
+    serviceName: "my-company"
+    spanAttributes:
+    - key: "company-key"
+      value: "company-value"
 ```
 
-```text
-...
-Status:
-  Conditions:
-     ...
-    Message:               parametersRef resource is resolved
-    Observed Generation:   1
-    Reason:                ResolvedRefs
-    Status:                True
-    Type:                  ResolvedRefs
+To change the telemetry configuration for a particular Gateway, reference the following NginxProxy in the Gateway's spec:
+
+```yaml
+apiVersion: gateway.nginx.org/v1alpha2
+kind: NginxProxy
+metadata:
+  name: gateway-telemetry-service-name
+  namespace: default
+spec:
+  telemetry:
+    exporter:
+      batchSize: 50
+      batchCount: 5
+    serviceName: "my-app"
+    spanAttributes:
+    - key: "app-key"
+      value: "app-value"
 ```
 
-If everything is valid, the `ResolvedRefs` condition should be `True`. Otherwise, you will see an `InvalidParameters` condition in the status.
+These NginxProxy resources are merged and the following settings are applied to the Gateway:
+
+```yaml
+  telemetry:
+    exporter:
+      endpoint: "my.telemetry.collector:9000"
+      interval: "60s"
+      batchSize: 50
+      batchCount: 5
+    serviceName: "my-app"
+    spanAttributes:
+    - key: "app-key"
+      value: "app-value"
+```
+
+#### Disable Tracing for a Gateway
+
+A GatewayClass references the following NginxProxy which configures telemetry:
+
+```yaml
+apiVersion: gateway.nginx.org/v1alpha2
+kind: NginxProxy
+metadata:
+  name: gateway-class-telemetry
+  namespace: default
+spec:
+  telemetry:
+    exporter:
+      endpoint: "my.telemetry.collector:9000"
+      interval: "60s"
+    serviceName: "my-company"
+```
+
+To disable tracing for a particular Gateway, reference the following NginxProxy in the Gateway's spec:
+
+```yaml
+apiVersion: gateway.nginx.org/v1alpha2
+kind: NginxProxy
+metadata:
+  name: gateway-disable-tracing
+  namespace: default
+spec:
+  telemetry:
+    disabledFeatures:
+    - DisableTracing
+```
+
+These NginxProxy resources are merged and the following settings are applied to the Gateway:
+
+```yaml
+telemetry:
+    exporter:
+      endpoint: "my.telemetry.collector:9000"
+      interval: "60s"
+    serviceName: "my-app"
+    disabledFeatures:
+    - DisableTracing
+```
 
 ---
 
-## Manually create the configuration
+## Configuring the GatewayClass NginxProxy on Install
+
+By default, the NginxProxy resource is not created when installing NGINX Gateway Fabric. However, you can set configuration options in the `nginx.config` Helm values, and the resource will be created and attached to the GatewayClass when NGINX Gateway Fabric is installed using Helm. You can also [manually create and attach](#manually-creating-nginxproxies) the resource after NGINX Gateway Fabric is already installed.
```

Comment:
I think this is technically going to change because we'll be including the nginx image information in the NginxProxy, which means by default we'll be creating it and attaching it to the GatewayClass at install time.

Though I can change this once that is actually implemented.